api/agentpb/agent.proto

```diff
@@ -339,18 +339,36 @@ message JobStatusResponse {
   bool alive = 1;
 }
 
+// S3LocationConfig represents S3 bucket configuration.
+message S3LocationConfig {
+  string endpoint = 1;
+  string access_key = 2;
+  string secret_key = 3;
+  string bucket_name = 4;
+}
+
 // StartJobRequest is a ServerMessage asking pmm-agent to start job.
 message StartJobRequest {
   // Echo is a simple echo job that can be delayed.
   message Echo {
     string message = 1;
     google.protobuf.Duration delay = 2;
   }
+  // MySQLBackup is job for backup MySQL service.
+  message MySQLBackup {
+    // DSN for the service.
+    string dsn = 1;
```

Comment:
I think it should be service_id, not `dsn`. `dsn` we can find on managed side by service id.

Reply:
Ah, wrong file. Nevermind.